web/apps/labelstudio/src/pages/WebhookPage/WebhookList.jsx

```diff
@@ -40,13 +40,17 @@ const WebhookList = ({ onSelectActive, onAddWebhook, webhooks, fetchWebhooks })
           <Block name="webhook-list">
             {webhooks.map((obj) => (
               <Elem key={obj.id} name="item">
-                <Elem name="item-active">
-                  <Toggle name={obj.id} checked={obj.is_active} onChange={onActiveChange} />
+                <Elem name="info-wrap">
+                  <Elem name="url-wrap">
```

Comment:
what are these wrappers? don't see them neigher in this PR or current code. am I missing something?

Reply:
The difference either way is empty div, or an unused class target on the component div. If we feel strongly about it we can always remove it, just not myself certain of whether it is used/needed. For now to unblock this PR I'm going to side with keep it, and ask @carly-bartel if we need it to be producing named class targets for these wrapping elements.

Reply:
yeah, okay, just didn't understand the reasoning behind this